terraform/modules/iot-edge/main.tf

```diff
@@ -31,15 +28,27 @@ resource "azurerm_public_ip" "iot_edge" {
 }
 
 resource "azurerm_network_security_group" "iot_edge" {
-  name                = "nsg-${local.dns_label_prefix}"
+  name                = "nsg-iotedge-${var.resource_uid}"
   resource_group_name = var.resource_group_name
   location            = var.location
 
   security_rule {
-    name                       = "nsgr-allow-22"
-    priority                   = 1000
+    name                       = "nsgsr-AllowBastionInbound"
+    priority                   = 900
     access                     = "Allow"
     direction                  = "Inbound"
+    protocol                   = "*"
+    source_address_prefix      = "10.0.2.0/26"
```

Comment:
Should this security rule also use the cidr function?

Reply:
Missed it - thanks!!